src/dlload.c

```diff
@@ -283,7 +283,8 @@ JL_DLLEXPORT void *jl_load_dynamic_library(const char *modname, unsigned flags,
     is_atpath = 0;
 
     JL_TIMING(DL_OPEN, DL_OPEN);
-    jl_timing_puts(JL_TIMING_CURRENT_BLOCK, modname);
+    if (flags & JL_RTLD_NOLOAD)
+        jl_timing_puts(JL_TIMING_CURRENT_BLOCK, modname);
```

Comment:
Don't we only want to do this when `!(flags & RTLD_NOLOAD)`?  E.g. only when we're actually loading something?

Reply:
Yes, I got this reversed 😞 

Good catch, thank you!